vnext/src/IntegrationTests/websocket_integration_test_server_binary.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const WebSocket = require('ws');
+
+console.log(`\
+WebSocket integration test server
+
+This will send each incoming message back, with the string '_response' appended.
+An incoming message of 'exit' will shut down the server.
```

Comment:
Nit: This comment does not align with the actual impl, which is commented out.

Reply:
I should have added a comment. It's "forked" from the Node WS server provided by facebook/RN.

This file may get polished and launched on CI loops, or deleted in favor of an in-process native server instance.